src/main/java/com/battleship/backend/models/Board.java

```diff
@@ -17,6 +17,9 @@ public Board(String name){
     }
 
     public void addShip(Ship ship, int row, int col){
+        System.out.println("ship:" + ship.getName());
+        System.out.println("row: " + row);
+        System.out.println("col: " + col);
```

Comment:
Was printing to the console part of your requirements? If it was for demo purposes from your mentors, carry on. If not, I'm not so sure this is necessary for the API.

Reply:
Accidentally committed before I removed those. Removed now.